### PR TITLE
Badges & coverage

### PR DESCRIPTION
**Pull request by RobertHebel**
Deployed to https://beghp.github.io/gh-pages-rc-10<br>Add new badges<br>Fix generating a coverage report<br><br><br>- [ ] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)<br>- [ ] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`<br><br>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,15 @@
...
 > Feature-rich, react-way carousel component that does not suck 
 
+[![Downloads](https://img.shields.io/npm/dm/@brainhubeu/react-carousel)](https://img.shields.io/npm/dm/@brainhubeu/react-carousel)
 [![CircleCI](https://circleci.com/gh/brainhubeu/react-carousel.svg?style=svg)](https://circleci.com/gh/brainhubeu/react-carousel)
+[![Last commit](https://img.shields.io/github/last-commit/brainhubeu/react-carousel.svg)](https://img.shields.io/github/last-commit/brainhubeu/react-carousel.svg)
+[![license](https://img.shields.io/npm/l/@brainhubeu/react-carousel.svg)](https://www.npmjs.com/package/@brainhubeu/react-carousel)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com)
+
+[![Activity](https://img.shields.io/github/commit-activity/m/brainhubeu/react-carousel.svg)](https://img.shields.io/bundlephobia/min/@brainhubeu/react-carousel?label=minified)
+[![Minified](https://img.shields.io/bundlephobia/min/@brainhubeu/react-carousel?label=minified)](https://img.shields.io/bundlephobia/min/@brainhubeu/react-carousel?label=minified)
```
Comment (piotr-s-brainhub) [inline README.md:12]: what about other badges?
- https://github.com/dwyl/hits
- https://github.com/all-contributors/all-contributors
-
```
[![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
```
- https://github.com/Naereen/badges#website-updown

Comment (piotr-s-brainhub) [reply README.md:12]: I'd like to have `NPM updated` badge but I cannot find it.

(this badge)[https://nodei.co/npm/mongoose.png] contains the time of the last NPM publish, possibly we could replace several NPM badges into this one as it contains more info but unfortunately, it doesn't display the number of NPM downloads (as I remember, it used to display that but later it broke)

Comment (RobertHebel) [reply README.md:13]: I added contributors and renovate badges. The number of hits isn't so impressive so I would like to skip this badge

Comment (RobertHebel) [reply README.md:13]: From what I see the nodei.co tool doesn't work with our repos 
Zrzut ekranu 2020-04-14 o 16 59 58

**Pull request closed without merging**